doc/api/buffer.md

```diff
@@ -4943,6 +4952,20 @@ added: v3.0.0
 
 An alias for [`buffer.constants.MAX_STRING_LENGTH`][].
 
+### `buffer.resolveObjectURL(id)`
+<!-- YAML
+added: REPLACEME
+-->
+
+> Stability: 1 - Experimental
+
+* `id` {string} A `'blob:node:...` URL string returned by a prior call to
+  `URL.createObjectURL()`.
+* Returns: {Blob}
+
+Resolves a `'blob:node:...'` an associated {Blob} object registered using
+a prior call to `URL.createObjectURL()`.
```

Comment:
we might want to change from `blob:node:` since it might cause some confusion. Blob URLs adopt origins from their trailing bits and so you often see things like on `example.com`:

```console
> URL.createObjectURL(new Blob(['hi'], {type:'text/javascript'}))
"blob:http://example.com/37488b44-4309-4128-b6f9-d62762fe0b35"
```

Just to something that won't collide / is a known carve out instead of just IDs. We should just document it basically so we have known expectations that even if it uses the `node:` protocol it cannot be retrieved with `import('node:...')`, it is true for browsers already but might confuse users.

Reply:
Good point. Switching to `blob:nodedata:...` for now. If there's a better suggestion let me know.